docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  chatdev:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    command: online_log/app.py
```

Comment:
Im more inclined to give the developer a shell rather than running the webapp. It might be more useful to comment this out and add the command `docker-compose run --service-ports chatdev /bin/bash` as the default instead to give the developer the freedom to still play around with the project